### PR TITLE
2022.12.1

### PR DESCRIPTION
**Pull request by tedpatrick**

### CODE REVIEW

**Review by FabioRosado — APPROVED** [review]

**Review by madhur-tandon — APPROVED** [review]

**Comment by FabioRosado** [comment]
Ah the version test needs to make the release level optional on the pattern

File: pyscriptjs/src/runtime.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ const logger = getLogger('pyscript/runtime');
 
 // VERSION
 // Version number of release
-export const version = '2022.12.1.RC1';
+export const version = '2022.12.1.release';
```
Comment (antocuni) [inline pyscriptjs/src/runtime.ts:9]: are we sure we want the `.release`?
It looks very weird to me, I'd expect just `2022.12.1` (i.e., the same string that we use in the URLs)

Comment (tedpatrick) [reply pyscriptjs/src/runtime.ts:9]: Here are @JeffersGlass notes...
"Sets the __version__ and version_info properties from provided JSON data
    Args:
        version_from_runtime (str): A "dotted" representation of the version:
            YYYY.MM.m(m).releaselevel
            Year, Month, and Minor should be integers; releaselevel can be any string
    """

Comment (tedpatrick) [reply pyscriptjs/src/runtime.ts]: I propose making the releaselevel optional in the regex tests.

Comment (tedpatrick) [reply pyscriptjs/src/runtime.ts]: Implemented and tests running

**Pull request merged**